debian/debathena-pyhesiodfs.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Hesiod automounter for Athena lockers
+Requires=openafs-client.service
+After=local-fs.target network.target openafs-client.service
+
+[Service]
+Type=simple
+Environment="pyhesiodfs_dir=/mit"
+OOMScoreAdjust=-1000
+ExecStartPre=/bin/mkdir -p /mit
```

Comment:
Is defined on line 8. `${pyhesiodfs_dir}` for systemd unit file is equivalent to quote-enclosed `"$pyhesiodfs_dir"` for shell.